response_operations_ui/templates/messages.html

```diff
@@ -166,30 +195,30 @@ <h1 class="u-fs-l u-mt-l">Oops! Something went wrong</h1>
                   "id": 'ru-ref-' + loop.index|string
                 },
                 {
-                  "value": message.business_name if message.business_name else 'Unavailable',
+                  "value": businessName,
```

Comment:
There is a chance the business name might not exist, is there a reason why you removed this?